docs/dev/table/tableApi.md

```diff
@@ -2474,17 +2474,30 @@ STRING.rpad(len INT, pad STRING)
         <p>Returns a string right-padded with the given pad string to a length of len characters. If the string is longer than len, the return value is shortened to len characters. E.g. "hi".rpad(4, '??') returns "hi??",  "hi".rpad(1, '??') returns "h".</p>
       </td>
     </tr>
+
     <tr>
       <td>
         {% highlight java %}
 STRING.fromBase64()
 {% endhighlight %}
       </td>
-
+      
```

Comment:
Try to avoid meaningless updates like that.

Reply:
OK, it is because of rebasing, will take care.